isis/src/qisis/objs/StretchTool/StretchTool.cpp

```diff
@@ -301,6 +311,13 @@ namespace Isis {
     layout->addWidget(m_stretchMaxEdit);
     layout->addWidget(advancedButton);
     layout->addWidget(m_flashButton);
+
+    // should only display if gray stretch
+    // Save/Restore strech only supported for Grayscale images. Hide buttons if in RGB.
```

Comment:
I'm omitting the change suggested in this comment, because I'm working on adding RGB now.